src/main/java/net/snowflake/ingest/streaming/internal/ParquetValueParser.java

```diff
@@ -77,7 +81,7 @@ static ParquetBufferValue parseColumnValueToParquet(
                   physicalType);
           value = intVal;
           stats.addIntValue(BigInteger.valueOf(intVal));
-          size = 4;
+          estimatedParquetSize = 4;
```

Comment:
what's the reason of using = here instead of +=?

Reply:
You are right, it should be `+=` everywhere. I had the `1 + 0.125` at the end and forgot to change all `+` to `+=` when moving that line to the top. Fixed it.